section4/README.md

```diff
@@ -50,27 +50,27 @@ Create a new Gist. In it, write a letter introducing yourself to your future ins
 
 - Introduce yourself and share your “why” for taking this step to change your career.
 - What are your strengths? What commitments can you make as far as what/how you will contribute to the cohort and classroom?
-- What do your instructors need to know about you (consider including your workoing styles, your values, how you like to receive feedback, anything else)?
+- What do your instructors need to know about you (consider including your working styles, your values, how you like to receive feedback, anything else)?
 - What did you learn about yourself, as a person and a student, during Mod 0?
 
 Make sure to include your full name and pronouns.
 
-- `Add Link to Gist that contains your letter - here`
+- https://gist.github.com/awirth224/2a9c1254843b93252833009991b65e22
 
 ## Mentorship Prep
 
 Mentorship is an integral part of the Turing experience and will help jumpstart your technical career. In order to get your mentor relationship started on the right foot, please complete the following deliverables:
 
-- [ ] [Create a profile on Turing Community Connect by following these directions](https://docs.google.com/document/d/1vpyKGu92l1HGkJzULNcyyE72946f4QO1DhQgIz3v1E0/edit?usp=sharing)
-- [ ] Complete the [Mentorship DTR Prep](https://gist.github.com/ericweissman/51965bdcbf42970d43d817818bfaef3c)
-  - [ ] Add link to your gist here: 
+- [x] [Create a profile on Turing Community Connect by following these directions](https://docs.google.com/document/d/1vpyKGu92l1HGkJzULNcyyE72946f4QO1DhQgIz3v1E0/edit?usp=sharing)
+- [x] Complete the [Mentorship DTR Prep](https://gist.github.com/ericweissman/51965bdcbf42970d43d817818bfaef3c)
+  - [x] Add link to your gist here: https://gist.github.com/awirth224/2f6f2a4f73a75da73c2b189b3d635b86
 
 ## Group Work Prep
 
 As part of Turing's project-based learning approach, you will often be working in pairs or larger groups. In order to set yourself (and your team) up for success, it is important to ensure you are prepared to be an equitable contributor and teammate.
 
-- [ ] Complete the [DTR Guiding Questions](https://gist.github.com/ericweissman/c56f3a98cdce761808c21d498a52f5c6)
-  - [ ] Add a link to your gist here: 
+- [x] Complete the [DTR Guiding Questions](https://gist.github.com/ericweissman/c56f3a98cdce761808c21d498a52f5c6)
+  - [x] Add a link to your gist here: https://gist.github.com/awirth224/3b4f693f506583a800d0b4ca79854f26
```

Comment:
Great baseline here! Definitely have this DTR template ready to pull up when you kick off your first Mod 1 project